src/index.html

```diff
@@ -6,12 +6,163 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form>
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="form_field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form_field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form_field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+            autocomplete="off"
+          >
+        </label>
```

Comment:
add one empty line between ALL same level elements

```suggestion
        <legend>Personal information</legend>
        
        <label class="form_field">
          Surname:
          <input
            type="text"
            name="surname"
            autocomplete="off"
          >
        </label>
        
        <label class="form_field">
          Name:
          <input
            type="text"
            name="name"
            autocomplete="off"
          >
        </label>
        
        <label class="form_field">
          How old are You?
          <input
            type="number"
            name="age"
            value="12"
            min="1"
            max="100"
            autocomplete="off"
          >
        </label>
```